csrc/stdio/pf_io_stdio.c

```diff
@@ -58,3 +58,7 @@ void sdTerminalTerm( void )
 {
 }
 
+cell_t sdSleepMillis(cell_t msec)
+{
+    return PF_ERR_NOT_SUPPORTED;
```

Comment:
This will cause MSEC to print an error message and then try to sleep by spinning the CPU. That is OK.